brainframe_qt/ui/brainframe_app.py

```diff
@@ -169,109 +163,33 @@ def _handle_error_(*args):
         # noinspection PyProtectedMember
         return QApplication.instance()._handle_error(*args)
 
-    # noinspection PyMethodMayBeStatic
-    def _init_server_settings(self):
-        api.set_url(self.server_config.server_url)
-
-        username = self.server_config.server_username
-        password = self.server_config.server_password
-
-        if username and password:
-            password = decrypt(password)
-            api.set_credentials((username, password))
+    def _init_config(self):
+        self.setOrganizationDomain('aotu.ai')
 
     # noinspection PyMethodMayBeStatic
     def _shutdown(self):
         api.close()
         gobject_init.close()
 
+        self.connection_manager.requestInterruption()
```

Comment:
Pretty cool that QThreads have this built in method for asking a thread to stop. This would have saved us the trouble of adding a `_running` parameter and a `close()` method to every single regular thread ever on the server.

Reply:
yep very convenient